specifyweb/frontend/js_src/lib/components/FormParse/cells.ts

```diff
@@ -355,6 +356,10 @@ export function parseFormCell(
       getBooleanAttribute(cellNode, 'invisible') !== true ||
       parsedCell === processCellType.Unsupported,
     ...parsedCell({ cell: cellNode, table, getProperty }),
+    isBold:
+      getBooleanAttribute(cellNode, 'isBold') === true ||
+      parsedCell === processCellType.Unsupported,
+    ...parsedCell({ cell: cellNode, table, getProperty }),
```

Comment:
parsedCell is not supposed to be call twice - after your change it's called twice